### ISSUE TITLE
TypeError: Too few arguments for TypedArray

### ISSUE THREAD

**Issue by junelsolis** [Labelbox/labelbox-python#377]
I encounter a problem when trying to `from labelbox import Client` within Python script. The environment is Python 3.9 and `labelbox==3.11.0`

__STEPS TO REPRODUCE__
- Create environment with `python=3.9`
- Install using pip: `labelbox==3.11.0` and `labelbox[data]`
- Run python script containing `from labelbox import Client`

The following traceback should be output:
```
...
  File "/home/junelsolis/mambaforge/envs/he-mucin-splinedist/lib/python3.9/site-packages/labelbox/data/serialization/labelbox_v1/feature.py", line 6, in <module>
    from ...annotation_types.types import Cuid
  File "/home/junelsolis/mambaforge/envs/he-mucin-splinedist/lib/python3.9/site-packages/labelbox/data/annotation_types/__init__.py", line 1, in <module>
    from .geometry import Line
  File "/home/junelsolis/mambaforge/envs/he-mucin-splinedist/lib/python3.9/site-packages/labelbox/data/annotation_types/geometry/__init__.py", line 3, in <module>
    from .mask import Mask
  File "/home/junelsolis/mambaforge/envs/he-mucin-splinedist/lib/python3.9/site-packages/labelbox/data/annotation_types/geometry/mask.py", line 9, in <module>
    from ..data import MaskData
  File "/home/junelsolis/mambaforge/envs/he-mucin-splinedist/lib/python3.9/site-packages/labelbox/data/annotation_types/data/__init__.py", line 1, in <module>
    from .raster import ImageData
  File "/home/junelsolis/mambaforge/envs/he-mucin-splinedist/lib/python3.9/site-packages/labelbox/data/annotation_types/data/raster.py", line 16, in <module>
    class RasterData(BaseModel, ABC):
  File "/home/junelsolis/mambaforge/envs/he-mucin-splinedist/lib/python3.9/site-packages/labelbox/data/annotation_types/data/raster.py", line 22, in RasterData
    arr: Optional[TypedArray[Literal['uint8']]] = None
TypeError: Too few arguments for TypedArray
```

**Comment by msokoloff1** [comment]
I'll look into this right now.

**Comment by msokoloff1** [comment]
It looks like in the latest numpy release they added support for typing np.ndarrays. So a quick fix for you would be to install an older version of numpy `pip install 'numpy<1.22.0' `. I'll work on a more permanent solution.

**Comment by junelsolis** [comment]
I can confirm `numpy<1.22.0` fixes the problem. Thank you.

**Comment by msokoloff1** [comment]
This issue has been fixed `3.11.1`. If you encounter any problems please let me know
